src/run.js

```diff
@@ -1,38 +1,81 @@
 const tsdLite = require('tsd-lite');
 const { formatTsdResults } = require('./formatter');
-const { fail } = require('./fail');
-const { pass } = require('./pass');
+
+/** @typedef {import('@jest/test-result').TestCaseResult} TestCaseResult */
 
 const TEST_TITLE = 'tsd typecheck';
 
+/**
+ * @param {'failed' | 'passed'} status
+ * @param {number} duration
+ * @param {string=} errorMessage
+ * @returns {TestCaseResult}
+ */
+function toJestResult(status, duration, errorMessage) {
+  return {
+    ancestorTitles: [],
+    failureDetails: [],
+    failureMessages: errorMessage ? [errorMessage] : [],
+    fullName: TEST_TITLE,
```

Comment:
Currently `fullName` is set to absolute path of the test file. Not sure if this is a breaking change, or just a fix?

Reply:
seems fine - we expose the filename via `testFilePath`